details.html

```diff
@@ -29,7 +29,7 @@
   </head>
   <body>
     <!-- Loader -->
-    <div class="loader">
+    <div class="loader" id="loader">
       <div class="spinner">
```

Comment:
Why do you need both class and id?

Reply:
because I used get element by id in the js.

Reply:
Cant you get it via class? I mean you already have a class loader.